added trouble shooting (#12)
baconcheese113/react-native-webrtc-minimal: README.md

## Next Steps

- **This repo is purposefully built for simple local-only calls. Not to demonstrate multiple devices, signalling servers, Janus integrations, etc...** But you can start here to understand the basics of react-native-webrtc. For more advanced things, try searching the posts in [the official discourse group](https://react-native-webrtc.discourse.group/)

> ## Trouble Shooting

Reviewer: ```suggestion
## Troubleshooting
```

> I am adding this in June 2020, I am on Windows so this might be behaving differently for other OS's 

Reviewer: ```suggestion
```

> - try using LTS, as of 06/20 it works with Node 12 (specifically v12.18.1)
> - if actually on Node v12.18.1 make sure you fixed [this bug they had](https://stackoverflow.com/questions/58120990/how-to-resolve-the-error-on-react-native-start)
> - change the gradle to 6.3 if running into problems that might help, I couldn't build without that. see [here](https://stackoverflow.com/questions/35000729/android-studio-could-not-initialize-class-org-codehaus-groovy-runtime-invokerhel) for refference 

Reviewer: ```suggestion
- Try using the [Active LTS version of Node](https://nodejs.org/en/about/releases/) 
- For Node v12 please be aware of this bug [this bug they had](https://stackoverflow.com/questions/58120990/how-to-resolve-the-error-on-react-native-start)
```
